cli/build.rs

```diff
@@ -0,0 +1,36 @@
+use std::process::{Command, ExitStatus};
+
+fn main() {
+    set_env("GIT_HASH", "git", &["rev-parse", "HEAD"]);
+
+    if !set_env(
```

Comment:
Unfortunately, this approach has a flaw. It gives the current tag name, if there is one. But we need the name of the tag or branch on which the build is taking place. Example: a commit is made and marked with a tag at the same time, GitHub starts 2 builds: by commit and by tag, in both cases the tag will appear in the version, which is not correct. 

You can keep this approach only as fallback, but the GitHub action should be the main source of the branch/tag name (the value specified in GitHub action should override the value estimated here). Another approach is just remove this code and always use env variables values set by GitHub action

Reply:
One more problem with this approach: it will not work when build is done within docker container. To set branch/tag name within docker it should be explicitly passed as an argument (as an example this happens here: https://github.com/qoollo/bob/blob/master/dockerfiles/alpine/Dockerfile#L7)

Reply:
If we remove all the code, there will be no version info on manual builds whatsoever. So it will be used as a fallback